pkg/util/utils.go

```diff
@@ -111,10 +111,20 @@ func (m MultiError) ToError() error {
 // EscapeStringForSSHUse will parse string and replace
 // shell special characters with escape
 func EscapeStringForSSHUse(s string) string {
-	r := strings.NewReplacer("$", "\\$",
+	r := strings.NewReplacer(
+		"$", "\\$",
 		`"`, `\"`,
 		"`", "\\`",
-		`\`, `\\`)
+		`\`, `\\`,
+		`(`, `\(`,
+		`)`, `\)`,
+		`{`, `\{`,
```

Comment:
The workaround mentioned here https://docs.openshift.org/latest/minishift/using/troubleshooting.html#special-characters-passwords is not working for me when I run

### 1. minishift start --username foo --password 'foo bar'
```
Registering machine using subscription-manager
E0512 13:52:02.292415   26563 start.go:176] Error starting the VM: Error registering the VM: ssh command error:
command : sudo -E subscription-manager register --auto-attach --username bgurung@redhat.com --password foo bar 
err     : exit status 64
output  : cannot parse argument: bar
. Retrying.
```

### 2. minishift start --username foo --password 'foo)bar'
```
Registering machine using subscription-manager
E0512 13:57:01.094153   27974 start.go:176] Error starting the VM: Error registering the VM: ssh command error:
command : sudo -E subscription-manager register --auto-attach --username bgurung@redhat.com --password foo(bar 
err     : exit status 1
output  : bash: -c: line 0: syntax error near unexpected token `('
bash: -c: line 0: `sudo -E subscription-manager register --auto-attach --username bgurung@redhat.com --password foo(bar '
. Retrying.
```

Same for other brackets too.
Semi colon seems to be not affected.